### ISSUE TITLE
Sleep Softlock

### ISSUE THREAD

**Issue by TatorThao** [SheerSt/pokewilds#257]
I found a softlock that puts me to sleep without being able to wake up. What I did was I went to the guide whilst in the sleeping bag. I haven't been able to do anything for the past 10 minutes, that encludes opening the menu, waking up, moving, etc. Luckily, when I left and reloaded the game it took me out of the sleeping bag.

**Comment by nousernamesavailable** [comment]
Was able to replicate this in a bed (using a mac)

**Comment by fourlastor** [comment]
@TatorThao @nousernamesavailable could you paste here the content of the file `error.log` in your pokewilds folder? That would be super helpful to track down this issue.

**Comment by nousernamesavailable** [comment]
@fourlastor Hi Danielle! Where does this get created?

**Comment by fourlastor** [comment]
@nousernamesavailable it should be in the same folder where the game is, check for a file named either "error" or "error.log"

**Comment by nousernamesavailable** [comment]
@fourlastor 
<img width="696" alt="image" src="https://user-images.githubusercontent.com/38391149/201472357-b05ad528-79ef-4d56-9f30-9cb2a23bb9ef.png"> 
yeah it's not coming up? I'm running on mac. Have checked that hidden files are visible. 

I am running the game through the jar file not the command line - something to do with how I've launched the game maybe?

**Comment by fourlastor** [comment]
@nousernamesavailable yes, that is definitely the issue. unfortunately mac puts pokewilds save folder in a temporary folder when ran like so, it should be in a path similar to this alert window:
![image](https://user-images.githubusercontent.com/1263058/202216636-4a416047-f252-4735-be16-7068024e6492.png)
 
Check the latest [release](https://github.com/SheerSt/pokewilds/releases/), it contains a readme with instructions on how to run the game correctly on mac
